package.json

```diff
@@ -51,7 +51,7 @@
     "deps:install": "npm ci",
     "deps:truncate": "npm prune --production",
     "typecheck": "tsc -p . --noEmit",
-    "test": "exit 0",
+    "test": "npx playwright test",
```

Comment:
the launch of the storybook should happen before the playwright tests

Reply:
If i run "npm run test" playwright runs the storybook before tests. I configured it in playwright.config (webServer property):
https://github.com/diplodoc-platform/components/pull/327/commits/437c10b0610ba829e3bd80b6640837324d49bcc0#diff-f679bf1e58e8dddfc6cff0fa37c8e755c8d2cfc9e6b5dc5520a5800beba59a92 
<img width="465" alt="image" src="https://github.com/user-attachments/assets/120aca76-a7f1-4630-abb0-d12556bf2e0d">